src/main/java/picard/util/IntervalListTools.java

```diff
@@ -271,26 +274,30 @@
             "by any of the input intervals. Will merge abutting intervals first. Output will be sorted.", optional = true)
     public boolean INVERT = false;
 
-    @Argument(doc = "What value (if anything) to output to stdout (for scripting)")
+    @Argument(doc = "What value to output to COUNT_OUTPUT file (for scripting)")
     public Output OUTPUT_VALUE = Output.NONE;
 
+    @Argument(doc = "File to which to print count of bases or intervals in final output interval list.  When this parameter is set OUTPUT_VALUE must be set to a value other than NONE.", optional = true)
+    public File COUNT_OUTPUT;
```

Comment:
if the default previously was that output goes to /dev/stdout, then perhaps we should leave it that way...

Reply:
That's fair, I have adjusted this so that by default the values are printed to standard out (and adjusted the command line validation to account for this as well).